tests/test_testing.py

```diff
@@ -47,3 +63,21 @@ def test_pytest_plugin_fail(pytester, book_list_html_response) -> None:
     save_fixture(base_dir, [book_list_html_response], item)
     result = pytester.runpytest()
     result.assert_outcomes(failed=1)
+
+
+class DateItemPage(WebPage):
+    async def to_item(self) -> dict:  # noqa: D102
+        return {
+            "foo": "bar",
+            "scraped_on": datetime.datetime.now().isoformat(),
+        }
```

Comment:
What do you think about using an actual item from zyte-common-items, to ensure we don't break it?

Reply:
zyte-common-items depends on web-poet, though it looks like it's not a problem for running tox